adcm_client/objects.py

```diff
@@ -661,6 +664,18 @@ def task_list(self, **args) -> "TaskList":
 
     def run(self, **args) -> "Task":
         with allure_step("Run action {}".format(self.name)):
+            config = self._get_config()
+
+            if 'config' in args:
+                pass
+            elif 'config_diff' in args:
+                for key, value in config.items():
+                    args['config_diff'].setdefault(key, value)
+                args['config'] = args.pop('config_diff')
+            else:
+                if config:
+                    args['config'] = config
```

Comment:
Do not use config var here, just call _get_config()

Reply:
Ok